controllers/watermarkpodautoscaler_controller.go

```diff
@@ -84,7 +84,7 @@ type WatermarkPodAutoscalerReconciler struct {
 // and what is in the WatermarkPodAutoscaler.Spec
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *WatermarkPodAutoscalerReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
+func (r *WatermarkPodAutoscalerReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
```

Comment:
you can now use the `ctx` in this function instead of `context.TODO()` 
for example here: https://github.com/DataDog/watermarkpodautoscaler/pull/78/files#diff-70d5250d0b22ca57247df02274172975492cd9862cc800a40ec91644a462d9ddR98

Reply:
Also line 88 is useless I think.

Reply:
thanks! PR updated